src/utils.ts

```diff
@@ -0,0 +1,321 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { loadFromModuleExportExpression } from '@graphql-mesh/utils';
+import type { YogaLogger } from 'graphql-yoga';
+import type { ImportFn } from '@graphql-mesh/types';
+import { default as Timeout } from 'await-timeout';
+import makeCancellablePromise from 'make-cancellable-promise';
+import fetch from 'node-fetch';
+import { HookFunction, HookStatus, Module, HookResponse, HookFunctionPayload } from './types';
+
+export interface MetaConfig {
+	logger: YogaLogger;
+	blocking: boolean;
+	baseDir: string;
+	importFn: ImportFn;
+}
+
+/**
+ * Import a module.
+ * @param modulePath Module path.
+ */
+export async function importFn(modulePath: string) {
+	return Promise.resolve(import(modulePath)).then(module => module);
```

Comment:
@kmaschi @nvyasadobe This function is translated to dynamic require in cjs. If the source code calling this function is esm you will not have any problem but if cjs code calls the dynamic require (cjs translation). For now I am updating the cjs folder every time I compile. 

We can separate this function to a new file and it should be in js so won't be compiled from ts to js